web/src/main/webapp/xslt/base-layout-cssjs-loader.xsl

```diff
@@ -281,6 +281,18 @@
       <script type="text/javascript" src="{$uiResourcesPath}lib/angular.ext/ui-ace.js?v={$buildNumber}"></script>
     </xsl:if>
 
+    <!-- Load variables that are required for user group profile management. -->
+    <xsl:if test="$angularApp = 'gn_admin'">
+      <script type="text/javascript">
+        var module = angular.module('<xsl:value-of select="$angularApp"/>');
+        module.config(['gnGlobalSettings',
+        function(gnGlobalSettings) {
+        gnGlobalSettings.isDisableLoginForm = <xsl:value-of select="$isDisableLoginForm"/>;
+        gnGlobalSettings.isUserProfileUpdateEnabled = <xsl:value-of select="$isUserProfileUpdateEnabled"/>;
+        gnGlobalSettings.isUserGroupUpdateEnabled = <xsl:value-of select="$isUserGroupUpdateEnabled"/>;
+        }]);
+      </script>
+    </xsl:if>
```

Comment:
If I build using -Denv=prod the logic does not seem to work.  But if I build with -Denv=dev then it works as expected. 
I'm questioning if this initialization is being done at the correct spot?  Or maybe I'm doing something incorrectly - maybe someone else can test this.

Reply:
I don't have such issue. I build with "mvn install -Denv=prod -T 1C -f pom.xml"

Here is the Keycloak mode:
![image](https://user-images.githubusercontent.com/74916635/123310694-5d752880-d4f4-11eb-9c06-3f36d313fd77.png)


and here is the local user mode:
![image](https://user-images.githubusercontent.com/74916635/123310665-5221fd00-d4f4-11eb-8dde-a03d58a72802.png)

Reply:
@wangf1122 
I rebooted my system and did a full clean build with -Denv=prod and it works now. So the problem was on my end. Sorry about that.
So this issue can be ignored.